docs/reference/run.md

```diff
@@ -0,0 +1,54 @@
+<!--[metadata]>
++++
+title = "run"
+description = "Runs a one-off command on a service."
+keywords = ["fig, composition, compose, docker, orchestration, cli,  run"]
+[menu.main]
+parent = "smn_compose_cli"
++++
+<![end-metadata]-->
+
+# run
+
+```
+Usage: run [options] [-e KEY=VAL...] SERVICE [COMMAND] [ARGS...]
+
+Options:
+--allow-insecure-ssl  Allow insecure connections to the docker
+                          registry
+-d                    Detached mode: Run container in the background, print
+                          new container name.
+--entrypoint CMD      Override the entrypoint of the image.
+-e KEY=VAL            Set an environment variable (can be used multiple times)
+-u, --user=""         Run as specified username or uid
+--no-deps             Don't start linked services.
+--rm                  Remove container after run. Ignored in detached mode.
+--service-ports       Run command with the service's ports enabled and mapped to the host.
+-T                    Disable pseudo-tty allocation. By default `docker-compose run` allocates a TTY.
+```
+
+Runs a one-time command against a service. For example, the following command starts the `web` service and runs `manage.py shell` in `python`. 
```

Comment:
"runs `manage.py shell` in `python`" feels a bit strange to me - makes it sound like `python` might be a container, or an image. Maybe:

> For example, the following command starts a container with the `web` service's image and configuration, and `python manage.py shell` as its command.

Also, I think `bash` would be a better example here, so people who don't know Django aren't thrown off wondering what `manage.py shell` is.

> For example, the following command starts a container with the `web` service's image and configuration, and `bash` as its command.
>
>     $ docker-compose run web bash